hrjobcontract/CRM/Hrjobcontract/Test/Fabricator/Contact.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class CRM_Hrjobcontract_Test_Fabricator_Contact {
+
+  private static $default = [
+    'contact_type' => 'Individual',
+    'first_name'   => 'John',
+    'last_name'    => 'Doe',
+    'sequential'   => 1
+  ];
+
+  public static function fabricate($params) {
+    $params                 = array_merge(self::$default, $params);
+    $params['display_name'] = "{$params['first_name']} {$params['last_name']}";
+
+    $result = civicrm_api3(
+      'Contact',
+      'create',
+      $params
+    );
+
+    return $result['values'][0];
```

Comment:
Both ways are fine for me. The first one is more compact and more JS-like while the second one seems to be easier to read (but longer). Also I personally use CRM_Utils_Array::first($result['values']) sometimes when I want to feel awesome (although it uses a foreach loop and returns the first iterated element instead of doing array_shift or something like that).